### PR TITLE
Added mask2CIDR() method to convert dotted decimal netmasks to CIDR n…

### PR DESCRIPTION
**Pull request by kika**
Hi, 

I've added a method which returns a CIDR length for an IPv4 address which is a valid IPv4 netmask. I use it to convert a pair of IPv4 address/IPv4 netmask into IPv4/CIDR notation.
Let me know what you think, if you want to change anything, etc. 
Thanks for the good library!

### CODE REVIEW

**Comment by kika** [comment]
Done. Not sure about 'null' part, I personally hate nulls, but doesn't really matter.

File: README.md [hunk 1]
```diff
@@ -164,6 +164,14 @@ var addr = ipaddr.parse("192.168.1.1");
 addr.octets // => [192, 168, 1, 1]
 ```
 
+`prefixLengthFromSubnetMask()` will return a CIDR prefix length for a valid IPv4 netmask or
+false if the netmask is not valid.
+
+```js
+ipaddr.IPv4.parse('255.255.255.240').prefixLengthFromSubnetMask() == 28
+ipaddr.IPv4.parse('255.192.164.0').prefixLengthFromSubnetMask()  == false
```
Comment (whitequark) [inline README.md]: Doc not updated.

File: test/ipaddr.test.coffee [hunk 2]
```diff
@@ -296,3 +296,40 @@ module.exports =
...
+    test.equal(ipaddr.IPv4.parse('255.128.0.0').prefixLengthFromSubnetMask(), 9)
+    test.equal(ipaddr.IPv4.parse('255.0.0.0').prefixLengthFromSubnetMask(), 8)
+    test.equal(ipaddr.IPv4.parse('254.0.0.0').prefixLengthFromSubnetMask(), 7)
+    test.equal(ipaddr.IPv4.parse('252.0.0.0').prefixLengthFromSubnetMask(), 6)
+    test.equal(ipaddr.IPv4.parse('248.0.0.0').prefixLengthFromSubnetMask(), 5)
+    test.equal(ipaddr.IPv4.parse('240.0.0.0').prefixLengthFromSubnetMask(), 4)
+    test.equal(ipaddr.IPv4.parse('224.0.0.0').prefixLengthFromSubnetMask(), 3)
+    test.equal(ipaddr.IPv4.parse('192.0.0.0').prefixLengthFromSubnetMask(), 2)
+    test.equal(ipaddr.IPv4.parse('128.0.0.0').prefixLengthFromSubnetMask(), 1)
+    test.equal(ipaddr.IPv4.parse('0.0.0.0').prefixLengthFromSubnetMask(), 0)
```
Comment (whitequark) [inline test/ipaddr.test.coffee]: No test for failure?

File: README.md (hunk 1 above)
Comment (kika) [inline README.md]: Ah, crap. Fixed.

File: test/ipaddr.test.coffee (hunk 2 above)
Comment (kika) [inline test/ipaddr.test.coffee]: I thought about it. While the success cases are limited, the failure cases are the rest 4 billion :-) How many you see reasonable? I can easily generate random IPs and test them. I did that from command line, but didn't include in the test itself.

Comment (whitequark) [inline test/ipaddr.test.coffee]: You have two paths that return null, so, two cases for those two paths.

Comment (kika) [inline test/ipaddr.test.coffee]: Done.

**Comment by whitequark** [comment]
Thanks!

**Pull request merged**